packages/cra-external-template/template/src/components/pages/login.tsx

```diff
@@ -0,0 +1,50 @@
+import * as React from 'react'
+import { Redirect } from 'react-router-dom'
+
+import Routes from '../../constants/routes'
+import { Button, Level } from '@reapit/elements'
+
+import * as loginStyles from './__styles__/styles'
+
+import logoImage from '../../assets/images/reapit-graphic.jpg'
+import connectImage from '../../assets/images/reapit-connect.png'
+import { AuthContext } from '../../context'
+import { redirectToLogin } from '@reapit/cognito-auth'
```

Comment:
Can we pre-integrate alias mapping to avoid relative import?

Reply:
No alias for external cra-template. Yield error when dev/build:
![image](https://user-images.githubusercontent.com/36730355/87667870-47fc6800-c795-11ea-805a-b1bec6445233.png)

Reply:
This will go with the new Auth Module